packages/react-rest/src/services/hatchifyReactRest/hatchifyReactRest.ts

```diff
@@ -60,42 +61,61 @@ export function hatchifyReactRest<TSchemaRecord extends SchemaRecord>(
   schemas: TSchemaRecord,
   dataSource: Source,
 ): ReactRest<TSchemaRecord> {
-  const storeKeys = Object.values(schemas).map((schema) => schema.name)
+  const storeKeys = Object.values(schemas).map((schema) =>
+    schemaNameWithNamespace(schema),
+  )
   createStore(storeKeys)
 
-  const newSchemas = Object.values(schemas).reduce((acc, schema) => {
-    acc[schema.name] =
+  const formattedSchemas = Object.values(schemas).reduce((acc, schema) => {
+    acc[schemaNameWithNamespace(schema)] =
       "displayAttribute" in schema ? schema : transformSchema(schema)
     return acc
   }, {} as Schemas)
 
-  const functions = Object.values(schemas).reduce((acc, schema) => {
-    acc[schema.name as SchemaKeys<TSchemaRecord>] = {
+  const models = Object.values(schemas).reduce((acc, schema) => {
+    const schemaName = schemaNameWithNamespace(schema)
+
+    acc[
+      schemaName as keyof ReactRest<TSchemaRecord> /* TODO: strong typing */
```

Comment:
@arthurpankiewicz type assertion here because `acc` can't be indexed with `string`. I assume this is something we'll take care of in V2? Lmk if you think it needs to be addressed in this PR, or if you'd like me to create a Jira ticket for it